libbeat/processors/actions/copy_fields.go

```diff
@@ -73,6 +73,7 @@ func (f *copyFields) Run(event *beat.Event) (*beat.Event, error) {
 		if err != nil && f.config.FailOnError {
 			logp.Debug("copy_fields", "Failed to copy fields: %s", err)
 			event.Fields = backup
+			event.PutValue("error.message", fmt.Sprintf("Failed to copy fields in copy_fields processor: %+v", err))
```

Comment:
You could use fmt.Errorf to create the error value and then reuse it here and on line 74? Like this the log and event error keep in sync.